### PR TITLE
Deprecate symfony/templating

### PR DESCRIPTION
**Pull request by kunicmarko20**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | yes
| License       | MIT

Deprecations for #114

### CODE REVIEW

File: tests/DependencyInjection/Compiler/TemplatingCompilerPassTest.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ class TemplatingCompilerPassTest extends AbstractTestCase
     private $compilerPass;
 
     /**
-     * {@inheritdoc}
+     * @group legacy
```
Comment (ElectricMaxxx) [inline tests/DependencyInjection/Compiler/TemplatingCompilerPassTest.php]: do we use that group somehow?

Comment (kunicmarko20) [reply tests/DependencyInjection/Compiler/TemplatingCompilerPassTest.php]: It is just to mark tests as deprecated.

Comment (kunicmarko20) [reply tests/DependencyInjection/Compiler/TemplatingCompilerPassTest.php]: Mainly it means, if any deprecation is thrown from this test method, ignore it.

Comment (ElectricMaxxx) [reply tests/DependencyInjection/Compiler/TemplatingCompilerPassTest.php]: so then lets see it as resolved :-)

**Pull request merged**